Documentation/Tech Stack.md

```diff
@@ -0,0 +1,26 @@
+# Tech Stack
+- The following frameworks are popular and widespread, so future developers will be able to pick up where we left off as requested by the client.
+
+- Website needs to be usable on desktop and mobile, many users may only have one or the other. 
```

Comment:
Mobile is not a concern that has been brought up by the client, though it is a valid one. But, I don't think we need to list this as a primary concern for our tech stack.

Reply:
Yeah, given that mobile first seems to be standard for web-design we can presume that any modern front end would support it.